tools/fast/series/series_createminerwithask.go

```diff
@@ -0,0 +1,27 @@
+package series
```

Comment:
strongly suggest we drop the stuttering and shorten filenames, eg series/createminer.go